### ISSUE TITLE
Glass Flip Panel

### ISSUE THREAD

**Issue by HugoBDesigner** [BEEmod/BEE2-items#2736]
An interesting idea for gel-based maps would be having a flip panel where one side is glass, the other is a portalable/unportalable surface where you can put gel on. It'd serve similar purposes to the glass-encased angled panel.

**Comment by ZombiePwner1** [comment]
@HugoBDesigner I'm not sure if this is actually possible.

**Comment by HugoBDesigner** [comment]
I don't see why not. Just "Start Enabled" to set if it starts glass up or down, and "Start Reversed" if it can't take the portalability of the wall.

**Comment by ZombiePwner1** [comment]
@HugoBDesigner I mean't like making one side of the panel glass and the other side portable.

**Comment by HugoBDesigner** [comment]
(Pardon the crappy rushed drawing)
![image](https://user-images.githubusercontent.com/6296061/42193160-613f2ea8-7e43-11e8-8080-fef94ad7d6fd.png)

**Comment by Endermage77** [comment]
Seems legit.

**Comment by Konclan** [comment]
It doesn’t need to be glass, it could just be teamspens anti-gel texture

**Comment by HugoBDesigner** [comment]
I considered the anti-gel textures, but I think glass is clearer and more intuitive, not to mention it lets you see if the other side is portalable or not

**Comment by SCIENTIST2004** [comment]
Maybe grating?
![42193160-613f2ea8-7e43-11e8-8080-fef94ad7d6fd](https://user-images.githubusercontent.com/35931028/42323994-97eae982-806a-11e8-8642-0a513c7a6814.png)

**Comment by HugoBDesigner** [comment]
Could be a possibility, but I personally find glass to be more intuitive

**Comment by Endermage77** [comment]
Plus grating is gel-permeable, so you could still paint the backside of the non-grating side.

**Comment by TeamSpen210** [comment]
Doesn't affect Conversion or Reflection Gel.

**Comment by HugoBDesigner** [comment]
What do you think would be better: blocking them too or not?

**Comment by ZipZoptheCat** [comment]
In theory the covered angled panel does this. But if you need to to be flat while it's gellable you may be out of luck.

**Comment by CTB33391** [comment]
Wait, what?
Fizzlers work as a brush entity (func_portal_cleanser something like that, it's been a while). Flip panels are a func_door_rotating, I imagine. Now I'm a complete hammer noob so please answer this for me: how can you tie a brush to 2 entities at the same time?
tl;dr don't think that's possible, sadly

**Comment by TeamSpen210** [comment]
Indeed you can't tie a brush to two entities. But you can set the fizzler to be parented to the `func_door_rotating`, which will make it follow the latter around when it spins. However I don't think it'll work quite correctly, I suspect the shader uses the position of the brushes to make the ripples and whatnot consistent. That'd mean moving the brush will cause those to shift pseudorandomly. Additionally I know the portal shot blocking ability of fizzlers doesn't correctly function for diagonal ones. Those both could be handled easily though by turning it off when rotating. I'd just use glass/grating though, there's no need for a fizzler (and the emitters would probably be too bulky to look good).

**Comment by HugoBDesigner** [comment]
I really don't think a fizzler would be appropriate for this. The sole purpose of this is for using with gel, blocking one side but not the other. It's mainly useful for Aperture Tag, but it could still be useful for Portal 2.

**Comment by TeamSpen210** [comment]
I'd just use glass, but then make the surface underneath entirely non-paintable (so it correctly behaves and doesn't allow gel to sneak under).

**Comment by Konclan** [comment]
Obsolete with surface lacquer

**Comment by HugoBDesigner** [comment]
> 
> 
> Obsolete with surface lacquer

Does surface lacquer adhere to flip panels?

**Comment by Konclan** [comment]
Should do

**Comment by TeamSpen210** [comment]
You can't right now specify that it should be only one side though, that's tricky.
